test/spec/plumbing.js

```diff
@@ -349,19 +350,17 @@ describe('Promise-Core\'s Plumbing', function () {
             };
             pl.callback.call(context, null, response, response.body);
 
-            context._rp_promise
+            return context._rp_promise
                 .then(function () {
-                    done(new Error('Expected promise to be rejected.'));
+                    throw new Error('Expected promise to be rejected.');
```

Comment:
I changed these to return promises instead of using `done` because mocha produces cleaner output if the test fails. With the `done` approach mocha just reports that the test timed out because one of the `expect`s throws an error and `done` is not called. I can change back to using `done` for consistency if you prefer.